### PR TITLE
DellEMC: S5232F support for show system-health command

### PR DESCRIPTION
**Pull request by arunlk-dell**
#### Why I did it
Support for show system-health command in s5232f
#### How I did it
Added the configuration, API changes to support system health
#### How to verify it
Execute "show system-health summary/detail/monitor-list" CLI.
[UnitTestlog.txt](https://github.com/Azure/sonic-buildimage/files/6930767/UnitTestlog.txt)

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

**Comment by zhangyanzhao** [comment]
@sujinmkang  please help to review

File: device/dell/x86_64-dellemc_s5232f_c3538-r0/system_health_monitoring_config.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "services_to_ignore": [],
+    "devices_to_ignore": ["fan.speed"],
+    "user_defined_checkers": [],
+    "polling_interval": 60,
+    "led_color": {
+        "fault"  : "amber",
+        "normal" : "green",
+        "booting": "blinking_green"
```
Comment (sujinmkang) [inline device/dell/x86_64-dellemc_s5232f_c3538-r0/system_health_monitoring_config.json:9]: what does "blinking_amber" indicate?

Comment (arunlk-dell) [reply device/dell/x86_64-dellemc_s5232f_c3538-r0/system_health_monitoring_config.json:9]: both "amber" and "blinking_amber" represents faulty state.

Comment (sujinmkang) [reply device/dell/x86_64-dellemc_s5232f_c3538-r0/system_health_monitoring_config.json:9]: @arunlk-dell does that mean the system_health daemon will set to amber as faulty state even though platform api can set to blinking_amber?

Comment (arunlk-dell) [reply device/dell/x86_64-dellemc_s5232f_c3538-r0/system_health_monitoring_config.json:9]: yes, system_health daemon checks whether the any of the entity is normal condition or faulty state and the led color is set based on it and for which we have mapped the amber and green color. The blinking_green is for bootup.
But the platform api supports 4 states as the system led is based on the two bit register value.

**Review by sujinmkang — APPROVED** [review]

**Pull request merged**